File: node.typescript/cart-backend/src/api/clientInfo.ts
import { SemVer } from 'semver';
import TaskVersion from '../cart/routes/taskVersion';


export default class ClientInfo {
    name: string;
    version: TaskVersion;
    constructor(name: string, version: string) {
        this.name = name || null;
        this.version = new TaskVersion(version);
    }
}

File: dropwizzard/cart-backend/src/main/java/com/mercadolibre/cart_backend/core/ClientInfo.java
package com.mercadolibre.cart_backend.core;

import com.github.zafarkhaja.semver.Version;

/**
 * Created by fvitali on 8/17/16.
 */
public class ClientInfo {
	private String name;
	private Version version;

	public ClientInfo(String name, Version version) {
		this.name = name;
		this.version = version;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public Version getVersion() {
		return version;
	}

	public void setVersion(Version version) {
		this.version = version;
	}
}
